### PR TITLE
Disable Xcode 10.3 CI builds

### PR DESCRIPTION
**Pull request by rcari**
There doesn't seem to be support for Xcode 10 anymore.

@brentleyjones

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,27 +8,27 @@ jobs:
...
+      - name: Build
+        run: swift build
+      - name: Test
+        run: swift test 2>&1 | xcpretty
+      - name: Gen fixtures
+        run: scripts/gen-fixtures.sh
+      - name: Check fixtures
+        run: scripts/diff-fixtures.sh
+      - name: Build fixtures
+        run: scripts/build-fixtures.sh
```
Comment (brentleyjones) [inline .github/workflows/ci.yml]: Can we remove the space changes here, and only have the `xcode`  version change? Thanks!

**Comment by brentleyjones** [comment]
@yonaskolb I'll let you see if 10.3 can be brought back.

**Comment by brentleyjones** [comment]
https://github.blog/changelog/2019-11-06-github-actions-macos-virtual-environment-updated-to-catalina

**Comment by yonaskolb** [comment]
Yeah it's a shame they removed Xcode 10.3. Instead of changing the Xcode version, we should change the Swift version used to compile

**Pull request merged**